nodebuilder/state/flags.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 var (
-	keyringAccNameFlag = "keyring.accname"
+	keyringAccNameFlag = "keyring.accnames"
```

Comment:
i'd prefer not to break the name of flag pls

it's okay to just have it as accname and allow for a list as it preserves backward compat